### PR TITLE
VB-3956 Exclude application reference from session window

### PR DESCRIPTION
**Pull request by AledWEvans**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt [hunk 1]
```diff
@@ -124,8 +124,14 @@ class OrchestrationSessionsController(private val visitSchedulerSessionsService:
       description = "Defaults to true if not passed. If true, will not return visit times that clash with higher priority legal or medical appointments.",
     )
     withAppointmentsCheck: Boolean? = true,
+    @RequestParam(value = "excludedApplicationReference", required = false)
+    @Parameter(
+      description = "The current application reference be to exclude from capacity count and double booking",
```
Comment (adaviesMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt:129]: Small update, change the description to:

"The current application reference to be excluded from capacity count and double booking"

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/enums/SessionConflict.kt [hunk 2]
```diff
@@ -3,5 +3,5 @@ package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.vi
 @Suppress("unused")
 enum class SessionConflict {
   NON_ASSOCIATION,
-  DOUBLE_BOOKED,
+  DOUBLE_BOOKING_OR_RESERVATION,
```
Comment (adaviesMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/enums/SessionConflict.kt:6]: has this been updated everywhere? Seems like the change is only here?

Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/enums/SessionConflict.kt:6]: Yes at the look of it,.. I assume the tests are not testing for it,.. you would only be testing the wiring anyhow, as there is no logic to test.

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**